### PR TITLE
Refactored ListSkills component

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Ref - #1291 

Changes: Refactored `ListSkills` component. Please suggest any other refactoring changes required in case I missed any.

Surge Deployment Link: https://pr-1509-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

File: src/components/Admin/ListSkills/ListSkills.js [hunk 1]
```diff
@@ -60,19 +63,18 @@ class ListSkills extends React.Component {
...
-      urls.API_URL +
-      `/cms/changeSkillStatus.json?model=${this.state.skillModel}&group=${
-        this.state.skillGroup
-      }&language=${this.state.skillLanguage}&skill=${
-        this.state.skillTag
-      }&reviewed=${this.state.skillReviewStatus}&editable=${
-        this.state.skillEditStatus
-      }&staffPick=${this.state.skillStaffPickStatus}&access_token=` +
-      cookies.get('loggedIn');
+    let self = this.state;
```
Comment (anshumanv) [inline src/components/Admin/ListSkills/ListSkills.js]: Destructure from state

File: src/components/Admin/ListSkills/ListSkills.js [hunk 2]
```diff
@@ -88,77 +90,57 @@ class ListSkills extends React.Component {
...
-      this.state.skillModel +
-      '&group=' +
-      this.state.skillGroup +
-      '&language=' +
-      this.state.skillLanguage +
-      '&skill=' +
-      this.state.skillName +
-      '&access_token=' +
-      cookies.get('loggedIn');
+    let self = this.state;
```
Comment (anshumanv) [inline src/components/Admin/ListSkills/ListSkills.js]: same

**Comment by Akshat-Jain** [comment]
@anshumanv Made the changes. Please review. Thanks :-)

**Comment by Akshat-Jain** [comment]
@akshatnitd I'm not sure how we can split this into smaller files, because this component is essentially just the table on the `Skills` tab in the Admin Panel. All the API calls required, and the other necessary functions should be placed alongside the main code for the table. Thanks :-)

**Pull request merged**